libs/src/services/solanaWeb3Manager/transactionHandler.ts

```diff
@@ -49,44 +55,68 @@ class TransactionHandler {
 
   /**
    * Primary method to send a Solana transaction.
-   *
-   * @typedef {Object} HandleTransactionReturn
-   * @property {Object} res the result
-   * @property {string} [error=null] the optional error
-   *  Will be a string if `errorMapping` is passed to the handler.
-   * @property {string|number} [error_code=null] the optional error code.
-   * @property {string} [recentBlockhash=null] optional recent blockhash to prefer over fetching
-   * @property {boolean} [skipPreflight=null] optional per transaction override to skipPreflight
-   * @property {any} [logger=console] optional logger
-   * @property {any} [feePayerOverride=null] optional fee payer override
-   *
-   * @param {Array<TransactionInstruction>} instructions an array of `TransactionInstructions`
-   * @param {*} [errorMapping=null] an optional error mapping. Should expose a `fromErrorCode` method.
-   * @param {Array<{publicKey: string, signature: Buffer}>} [signature=null] optional signatures
-   * @returns {Promise<HandleTransactionReturn>}
-   * @memberof TransactionHandler
    */
-  async handleTransaction ({ instructions, errorMapping = null, recentBlockhash = null, logger = console, skipPreflight = null, feePayerOverride = null, sendBlockhash = true, signatures = null, retry = true }) {
-    let result = null
+  async handleTransaction({
+    instructions,
```

Comment:
We may want to type the return explicitly here; looks like res is `any`

(Actually, probably want to do so further down in the called functions)

Reply:
see if you like the update